2 Niagara Fundamentals Exercise - Multiplexer/multiplexer-rt/src/com/tridiumuniversity/multiplexer/BMultiplexer.java

```diff
@@ -0,0 +1,257 @@
+package com.tridiumuniversity.multiplexer;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+import javax.baja.sys.BBoolean;
+
+import javax.baja.sys.BComponent;
+import javax.baja.sys.Flags;
+
+@NiagaraType
+
+@NiagaraProperty(
+    name = "ln0",
+    type = "BInteger",
+    defaultValue = "BInteger.make(0)",
+    flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "ln1",
+        type = "BInteger",
```

Comment:
Using `BInteger` for your inputs and your output results in a working component, but technically the assignment asked that they be `BBoolean` values instead.